IdentityCore/src/requests/sdk/MSIDBrokerResponseHandler.m

```diff
@@ -154,6 +157,16 @@ - (MSIDTokenResult *)handleBrokerResponseWithURL:(NSURL *)response sourceApplica
         return nil;
     }
     
+#if !AD_BROKER
+    // Save client flights if available
+    if (brokerResponse.clientFlights)
+    {
+        MSIDBrokerFlightProvider *flightProvider = [[MSIDBrokerFlightProvider alloc] initWithBase64EncodedFlightsPayload:brokerResponse.clientFlights];
+        
+        [MSIDFlightManager sharedInstance].flightProvider = flightProvider;
```

Comment:
ditto.

Also if moving the logic into a lower leverl, may not need to duplicate

Reply:
Updated